.travis.yml

```diff
@@ -40,14 +40,13 @@ env:
       CACHED_BUILD_DIR="$HOME/sklearn_build_oldest"
     # This environment tests the newest supported anaconda env
```

Comment:
I didn't get it, where should I add it? Are you asking to add it in the conda, python 2.7 distrib?